recipes/wslink/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "wslink" %}
+{% set version = "1.8.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wslink-{{ version }}.tar.gz
+  sha256: 3dbf76753807828f628c4fb57da27350af201c577670445d4748af47e4af7af9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - aiohttp <4
+
+test:
+  imports:
+    - wslink
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/kitware/wslink
+  summary: Python/JavaScript library for communicating over WebSocket
+  license: BSD-3-Clause
+  license_file: src/wslink/LICENSE
+
+extra:
+  recipe-maintainers:
+    - banesullivan
+    - jourdain
+    - kwrobot
```

Comment:
@kwrobot do agree to be a maintainer here.

Reply:
I just looked, that looks like a Kitware robot account.

Reply:
Indeed, this is an account we use to ensure Kitware as an organization has access to our repositories, etc. For precedence, we've added this account on other feedstocks

Reply:
FYI https://github.com/conda-forge/staged-recipes/pull/18785#discussion_r860123347